### PR TITLE
Add warning about broken Docker hosts in container entrypoint.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

Provides a general warning to catch the case encountered in https://github.com/netdata/netdata/issues/11933 and provide info to the user about it in the container logs.

##### Test Plan

n/a

##### Additional Information

Relevant to (but does not fix): https://github.com/netdata/netdata/issues/11933

### CODE REVIEW

**Review by iigorkarpov — DISMISSED** [review]

File: packaging/docker/run.sh [hunk 1]
```diff
@@ -9,14 +9,27 @@
...
+    echo >&2 "WARNING: To resolve this, please do one of the following:"
+    echo >&2 "WARNING:     * Update to a newer version of Docker and libseccomp (NOTE: This is the only possible resolution for Docker Swarm users)"
+    echo >&2 "WARNING:     * Download the default seccomp profile from https://raw.githubusercontent.com/moby/moby/master/profiles/seccomp/default.json, replace 'SCMP_ACT_ERRNO' on line 2 with 'SCMP_ACT_TRACE', and then add '--security-opt=seccomp=default.json' to the options being passed to 'docker run' when creating the netdata container."
+    echo >&2 "WARNING:     * Add '--security-opt=seccomp=unconfined' to the options being used to run the Netdata container (WARNING: This will significantly reduce the security of the container itself.)."
+  else
+    echo >&2 "ERROR: This container must be started in such a way that the entrypoint is runnning as EUID 0 inside the container. Current EUID is ${EUID}."
+    echo >&2 "ERROR: If you want the agent to run as a non-default user ID after the container is initialized, please set DOCKER_USR in the environment to the desired user name instead."
+    exit 1
```
Comment (ilyam8) [inline packaging/docker/run.sh:22]: This breaks https://github.com/netdata/netdata/pull/6543. I don't know if that functionality works tho 🤷‍♂️

Comment (Ferroin) [reply packaging/docker/run.sh]: I had orgotten that we actually did add support for that case.

I don’t think claiming works properly there, but I’m not sure.

Comment (Ferroin) [reply packaging/docker/run.sh]: Either way, I’ve removed the error case for now.

**Review by ilyam8 — DISMISSED** [review]

**Comment by ilyam8** [comment]
@Ferroin added related documentation in #12192, perhaps we just add a link to the docs instead of duplicating text?

**Comment by Ferroin** [comment]
> @Ferroin added related documentation in #12192, perhaps we just add a link to the docs instead of duplicating text?

Indeed, though we probably want to wait for that to be merged first so we have a guaranteed stable link to use here.

**Comment by ilyam8** [comment]
@Ferroin the link: https://learn.netdata.cloud/docs/agent/claim#known-issues-on-older-hosts-with-seccomp-enabled

**Comment by Ferroin** [comment]
Rebased to pick up CI changes.

Also added change suggested by @ilyam8.

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**